src/bindings/python/CMakeLists.txt

```diff
@@ -411,3 +412,80 @@ if(OpenVINODeveloperPackage_FOUND)
 
     ov_cpack(${OV_CPACK_COMPONENTS_ALL})
 endif()
+
+#if(DEFINED ENV{CI} OR DEFINED ENV{TF_BUILD} OR DEFINED ENV{JENKINS_URL})
+if(TRUE)
+    #
+    # Generate and check .pyi stub files generated by pybind11-stubgen vs commited files
+    #
+    execute_process(COMMAND ${Python3_EXECUTABLE} -m pip show pybind11-stubgen
+                    OUTPUT_VARIABLE pybind11_stubgen_info OUTPUT_STRIP_TRAILING_WHITESPACE
+                    RESULT_VARIABLE pybind11_stubgen_result)
+
+    if(pybind11_stubgen_result)
+        message(FATAL_ERROR "pybind11-stubgen is not installed. Please install it using 'pip install pybind11-stubgen'.")
+    else()
+        string(REGEX MATCH "Version: ([\\.0-9]+)" pybind11_stubgen_version "${pybind11_stubgen_info}")
+        set(pybind11_stubgen_version ${CMAKE_MATCH_1})
+        message(STATUS "pybind11-stubgen version: ${pybind11_stubgen_version}")
+    endif()
+
+    if(OpenVINODeveloperPackage_FOUND)
+        ov_check_pip_package(REQUIREMENT "pybind11-stubgen"
+                            RESULT_VAR pybind11_stubgen_AVAILABLE
+                            WARNING_MESSAGE "Please, install pybind11-stubgen if you plan to develop Python OpenVINO API"
+                            MESSAGE_MODE WARNING)
+    endif()
+    # Generate reference .pyi files for OpenVINO Python API
+    if(WIN32)
+        set(stub_gen_script "${CMAKE_CURRENT_SOURCE_DIR}/scripts/generate_pyapi_stubs.bat")
+    else()
+        set(stub_gen_script "${CMAKE_CURRENT_SOURCE_DIR}/scripts/generate_pyapi_stubs.sh")
+    endif()
+    file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/pyapi_stubs_generated)
+    message(STATUS "Script path: ${stub_gen_script}")
+    execute_process(
+        COMMAND ${CMAKE_COMMAND} -E 
+            env PYTHONPATH=${OpenVINOPython_SOURCE_DIR}/src:$ENV{PYTHONPATH}
+            env LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}:${OpenVINO_BINARY_DIR}/bin/intel64/Debug:$ENV{LD_LIBRARY_PATH}
```

Comment:
I think you need to replace `${OpenVINO_BINARY_DIR}/bin/intel64/Debug` with more robust code like `$<TARGET_FILE_FIR:openvino>`

Reply:
I removed `LD_LIBRARY_PATH` as a whole.